src/funcx_common/tasks/protocol.py

```diff
@@ -13,3 +13,5 @@ class TaskProtocol:
     task_id: str
     endpoint: t.Optional[str]
     status: TaskState
+    result: t.Optional[str]
+    result_reference: t.Optional[t.Dict[str, str]]
```

Comment:
Since this may be an arbitrary JSON dict in the future, I think the preferable type is
```python
t.Optional[t.Dict[str, t.Any]]
```

Reply:
Fixed.